src/components/common/dialogs/ExplanationDialog.js

```diff
@@ -113,12 +114,12 @@ const mapStylesToProps = () => ({
     width: '100%',
     marginBottom: getDesignRelativeHeight(theme.sizes.defaultDouble, false),
   },
-  label: {
```

Comment:
add some default label styles. as we have for a timestamp in this popup or in claim/sent/received feed popups